tests/e2e/helper_test.go

```diff
@@ -168,6 +168,11 @@ func setupDSCInstance(name string) *dscv1.DataScienceCluster {
 						ManagementState: operatorv1.Removed,
 					},
 				},
+				ModelRegistry: modelregistry.ModelRegistry{
+					Component: components.Component{
+						ManagementState: operatorv1.Managed,
```

Comment:
```suggestion
						ManagementState: operatorv1.Removed,
```

Reply:
So, just disable it in the e2e test, right?

Reply:
yes, dont think we need to set it as default in e2e, since it is not GA yet

Reply:
No, we need it enabled in the e2e test. Otherwise how will we know that the component reconcile works in rhoai 2.14?